spring2/src/main/java/com/unitbv/multipleConfigs/ConfigA.java

```diff
@@ -1,7 +1,12 @@
 package com.unitbv.multipleConfigs;
 
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 public class ConfigA {
+    @Bean("beanA")
```

Comment:
nu  e obligatoriu sa fie specificat numele intre paranteze. Comportamentul default este ca beanul va avea acelasi nume cu numele metodei - in cazul tau, va fi tot `beanA`